Formula/brainfuck.rb

```diff
@@ -0,0 +1,27 @@
+class Brainfuck < Formula
+  desc "Interpreter for brainfuck written in C"
+  homepage "https://github.com/fabianishere/brainfuck"
+  url "https://github.com/fabianishere/brainfuck/archive/2.7.0.tar.gz"
+  sha256 "445e00ef848f8b9a3ac101872ac08046b32b3477520ff06804d0ede5c3654c63"
+  head "https://github.com/fabianishere/brainfuck.git"
+
+  option "with-debug", "Compile interpreter with debug support"
+
+  depends_on "cmake" => :build
+
+  def install
+    args = std_cmake_args
+    args << "-DENABLE_DEBUG=ON" if build.with? "debug"
+
+    system "cmake", ".", *args
+    system "make"
+    bin.install "brainfuck"
+    man1.install "man/brainfuck.1"
+    lib.install "libbrainfuck.a"
+    doc.install "examples"
+  end
+
+  test do
+    system "#{bin}/brainfuck | cat"
```

Comment:
If there's no real test possible here just do something like:
```ruby
assert_match version.to_s, shell_output("#{bin}/brainfuck --version")
```
If you're installing the examples though maybe it's possible to do something real with those?

Reply:
Testing on one of the examples sounds like a good plan! I just copied this test from the previous but removed brainfuck formula, but didn't know the test wasn't thorough enough.

Reply:
Aye, standards have been slowly increased in the core over time. Any test is better than no test but ideally all tests would be functionality tests beyond simple things like checking the executable actually runs without panicking/etc.